packages/block-library/src/post-terms/variations.js

```diff
@@ -14,6 +14,15 @@ const variations = [
 		attributes: { term: 'category' },
 		isActive: ( blockAttributes ) => blockAttributes.term === 'category',
 	},
+
+	{
+		name: 'post_tag',
+		title: __( 'Post Tags' ),
+		description: __( "Display a post's tags." ),
+		icon,
```

Comment:
We need to set `tag` icon here.

Reply:
We have both a `category` and `post-categories` icons, we probably need the same for tags which we don't have right now. Maybe a good design follow-up.

Reply:
I've set a reminder to follow up here.